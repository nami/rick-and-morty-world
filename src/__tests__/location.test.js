import React from "react";
import { create } from "react-test-renderer";
import Location from '../components/location'

describe("Location component", () => {
  test("Matches the snapshot", () => {
    const location = create(<Location />);
    expect(location.toJSON()).toMatchSnapshot();
  });
});