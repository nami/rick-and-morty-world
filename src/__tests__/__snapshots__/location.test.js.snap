// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location component Matches the snapshot 1`] = `
<div
  className="location box has-background-link has-text-white animate__animated animate__slideInUp"
  onClick={[Function]}
>
  <img
    alt="a rick and morty world"
    src="1.jpg"
  />
  <div
    className="desc"
  >
    <p />
    <p />
  </div>
</div>
`;
